app/roles/throughput-cache.lua

```diff
@@ -0,0 +1,100 @@
+local log = require('log')
+
+local function weather_forecast_add(forecast)
+    log.info('weather forecast add')
+
+    box.space.weather_forecast:insert({
+        forecast.longitude,
+        forecast.latitude,
+        forecast.bucket_id,
+        forecast.raw_data,
+    })
+end
+
+local function weather_forecast_get(longitude, latitude)
+    log.info('weather forecast get')
+
+    local raw_forecast = box.space.weather_forecast:get({ longitude, latitude })
+    if not raw_forecast then
+        return false
+    end
+
+    return forecast_tuple_to_map(raw_forecast)
+end
+
+function forecast_tuple_to_map(raw_forecast)
```

Comment:
and there also is a method to cast to map from tuple - see [docs](https://www.tarantool.io/ru/doc/latest/reference/reference_lua/box_tuple/tomap/)

Reply:
and it is also preferable to use `local` to keep global namespace clean

Reply:
fixed